packages/flutter_tools/bin/fuchsia_tester.dart

```diff
@@ -130,13 +130,20 @@ Future<Null> run(List<String> args) async {
       // collector expects currentDirectory to be the root of the dart
       // package (i.e. contains lib/ and test/ sub-dirs).
       fs.currentDirectory = testDirectory.parent;
-      if (!await
-          collector.collectCoverageData(argResults[_kOptionCoveragePath]))
+      if (!await collector.collectCoverageData(argResults[_kOptionCoveragePath]))
         throwToolExit('Failed to collect coverage data');
     }
   } finally {
-    tempDirectory.deleteSync(recursive: true);
+    // This should not be necessary, but it turns out that
```

Comment:
This is never run on Windows, so this block can be removed.

Reply:
done